kilkari/src/main/java/org/motechproject/nms/kilkari/util/Timer.java

```diff
@@ -0,0 +1,64 @@
+package org.motechproject.nms.kilkari.util;
```

Comment:
maybe in a separate module called performance? (assuming that you don't run into circular dependencies)